### ISSUE TITLE
v 4.5.3.1  When editing field with primary key: "table does not contain a unique column" (OS X)

### ISSUE THREAD

**Issue by sirchuck** [phpmyadmin/phpmyadmin#11868]
It seems if my table name is something like "MY_TABLE" I can't edit the fields without getting the error. 

"my_table" allows me to edit fields without the error because it's lower case apparently. I am sending a dump of my database table below for you to test changing row fields from the table browser. 

Database Dump for you to recreate the error:

```sql
-- phpMyAdmin SQL Dump
-- version 4.5.3.1
-- http://www.phpmyadmin.net
--
-- Host: localhost
-- Generation Time: Jan 18, 2016 at 01:06 AM
-- Server version: 5.6.26
-- PHP Version: 5.5.30

SET SQL_MODE = "NO_AUTO_VALUE_ON_ZERO";
SET time_zone = "+00:00";


/*!40101 SET @OLD_CHARACTER_SET_CLIENT=@@CHARACTER_SET_CLIENT */;
/*!40101 SET @OLD_CHARACTER_SET_RESULTS=@@CHARACTER_SET_RESULTS */;
/*!40101 SET @OLD_COLLATION_CONNECTION=@@COLLATION_CONNECTION */;
/*!40101 SET NAMES utf8mb4 */;

--
-- Database: `Garbage`
--

-- --------------------------------------------------------

--
-- Table structure for table `GA_TABLE`
--

CREATE TABLE `GA_TABLE` (
  `id` char(10) NOT NULL,
  `t1` text NOT NULL,
  `t2` text NOT NULL,
  `t3` text NOT NULL
) ENGINE=MyISAM DEFAULT CHARSET=latin1;

--
-- Dumping data for table `GA_TABLE`
--

INSERT INTO `GA_TABLE` (`id`, `t1`, `t2`, `t3`) VALUES
('g1', 'g2', 'g2', 'g2'),
('g2', 'as', 'as', 'as'),
('g3', 'gag', 'ag', 'ag');

--
-- Indexes for dumped tables
--

--
-- Indexes for table `GA_TABLE`
--
ALTER TABLE `GA_TABLE`
  ADD PRIMARY KEY (`id`);

/*!40101 SET CHARACTER_SET_CLIENT=@OLD_CHARACTER_SET_CLIENT */;
/*!40101 SET CHARACTER_SET_RESULTS=@OLD_CHARACTER_SET_RESULTS */;
/*!40101 SET COLLATION_CONNECTION=@OLD_COLLATION_CONNECTION */;
```

**Comment by lem9** [comment]
Is the server running on OS X?

**Comment by sirchuck** [comment]
Yes, OS X Yosemite. Is this a file system case-sensitivity issue? I have a Mac OS Exteneded (Journaled) File System, not the case-sensitive one.

**Comment by lem9** [comment]
The issue looks similar to https://github.com/phpmyadmin/phpmyadmin/issues/11816; what's the value of your MySQL `lower_case_table_names` setting?

**Comment by sirchuck** [comment]
Apologies if this was already covered, I couldn't find an answer in my original search. Anyway I did go in and change the settings for lower_case_table_names to 0, 1, and 2.

My results:
0: Tables/Databases in capitols wouldn't load at all
1 & 2: Created my original problem. 

I didn't have a my.cnf file before this, so MySQL used it's defaults. 

I changed my database names to lower case now, but I wanted to let them know of the potential bug.
